miqa/core/tasks.py

```diff
@@ -137,7 +137,9 @@ def import_data(project_id: Optional[str]):
             else:
                 with open(import_path) as fd:
                     buf = fd.read()
-            import_dict = import_dataframe_to_dict(pandas.read_csv(StringIO(buf)))
+            import_dict = import_dataframe_to_dict(
+                pandas.read_csv(StringIO(buf)).fillna('').astype('string')
```

Comment:
I've been using `pandas.read_csv(StringIO(bug), na_filter=False)`. I'm not entirely sure what these flags do though, so I'm not sure if that is equivalent or not.